.rhdh/docker/Dockerfile

```diff
@@ -324,6 +336,8 @@ COPY --chown=1001:1001 $EXTERNAL_SOURCE_NESTED/packages/backend/src/instrumentat
 RUN chmod a=r ./instrumentation.js
 
 ENV NODE_OPTIONS="--no-node-snapshot"
-ENTRYPOINT ["node", "--require", "./instrumentation.js", "packages/backend", "--config", "app-config.yaml", "--config", "app-config.example.yaml", "--config", "app-config.example.production.yaml"]
+
+# Use bash -l -c to ensure a login shell is invoked, which sources /etc/profile.d/
+ENTRYPOINT ["bash", "-l", "-c", "node --require ./instrumentation.js packages/backend --config app-config.yaml --config app-config.example.yaml --config app-config.example.production.yaml"]
 
 # append Brew metadata here
```

Comment:
we could also set these new env vars as container ENVs here: https://gitlab.cee.redhat.com/rhidp/rhdh/-/blob/rhdh-1-rhel-9/build/ci/sync-midstream.sh?ref_type=heads#L701-702

that way you can compute them with `skopeo inspect` rather than needing to fetch and run the container.

Reply:
applied in https://gitlab.cee.redhat.com/rhidp/rhdh/-/commit/ca47ebea2b3817dc8a751a1639d6d087e07271f1

Reply:
So, now I can directly get the values using `skopeo inspect`?